routes/api/follows.js

```diff
@@ -4,24 +4,34 @@ const createError = require('http-errors');
 const { sequelize, Follow } = require("../../db/models");
 const {asyncHandler} = require('../utils');
 
+/* Delete a follower from a user */
+router.delete('/follows/:id', asyncHandler(async (req, res) => {
 
+        /* Initialize follow object */
+        const follow = await Follow.findPk(req.params.id);
 
+        /* Set status 404 if follow doesn't exist */
+        if (typeof follow === "undefined" ) {res.status(404)};
 
+        if (!res.locals.authenticated || res.locals.user.id !== follow.followerId) {
+            /* Return error 401 if
+                   user is not authenticated, or
+                   user is not the follower */
+            return next(createError(401));
 
-router.delete('/follows/:id', asyncHandler(async (req, res) => {
-    const follow = await Follow.findPk(req.params.id);
-    if (typeof follow === "undefined" ) {res.status(404)};
-
-    if (!res.locals.authenticated || res.locals.user.id !== follow.followerId) {
-        return next(createError(401)); 
-     } else if (res.locals.user.id === follow.followId && res.locals.authenticated) {
-    
-        await Follow.destroy({where:{id: req.params.id}})
+         } else if (res.locals.user.id === follow.followId && res.locals.authenticated) {
         
-     }
-
-    res.json({message: "Follower deleted"});
-
-}))
+            /* Destroy follow if
+                   user is authenticated, and
+                   user is follower */
+            await Follow.destroy({where:{id: req.params.id}})
```

Comment:
Since we have the `Follow` we want to delete (`follow`) we can just do `await follow.destroy()`

Reply:
`await Follow` has been replaced with the previously declared `follow`.